roles/openshift_facts/library/openshift_facts.py

```diff
@@ -640,25 +645,25 @@ def get_local_facts_from_file(filename):
 
 
 class OpenShiftFactsUnsupportedRoleError(Exception):
-    """OpenShift Facts Unsupported Role Error"""
+    """Facts Unsupported Role Error"""
```

Comment:
Not sure I am digging the error message output here...  maybe we need to genericize the facts module to Atomic Facts instead of OpenShift Facts?